docs-src/docs/kognic-io/calibrations/cameras-custom.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Custom Camera Calibrations
+---
+
+If your calibration is not in the list of supported standard camera calibrations, you can provide a custom calibration. The table below
+show the attributes of the `CustomCameraCalibration` object. The `wasm_base64` attribute is a base64 encoded string of the
+WebAssembly module that implements the calibration. The `test_cases` attribute is a list of `TestCase` objects that are used
+to validate the calibration when during creation. It is recommended to provide a few test cases to make sure that the
+calibration is correct. The `CustomCameraCalibration` object can conveniently be instantiated directly from the binary or 
+a wasm-file:
+```python
+calibration_file = CustomCameraCalibration.from_file("/path/to/calibration.wasm", ...)
+calibration_binary = CustomCameraCalibration.from_bytes(b"...", ...)
+```
+
+We provide a set of utilities that will make it easier to work with the WebAssembly module. More specifically we provide
+validation code and compilation code from a few different languages to WebAssembly. The code is available both as python
+functions and via the `kognicutil` cli.
+
+| Key                   | Value                         | Parameters         |
```

Comment:
Move this table to come directly after "The table below
show the attributes of the `CustomCameraCalibration` object."